Frends.AmazonS3.ListObjects/Frends.AmazonS3.ListObjects.Test/UnitTests.cs

```diff
@@ -0,0 +1,126 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using Frends.AmazonS3.ListObjects.Definitions;
+
+namespace Frends.AmazonS3.ListObjects.Test
+{
+    [TestClass]
+    public class ListObjectsTest
+    {
+        private readonly string _accessKey = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_AccessKey");
+        private readonly string _secretAccessKey = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_SecretAccessKey");
+        private readonly string _bucketName = Environment.GetEnvironmentVariable("HiQ_AWSS3Test_BucketName");
+
+        Source _source;
+        Options _options;
+
+
+        /// <summary>
+        /// List all objects.
+        /// </summary>
+        [TestMethod]
+        public void ListAll()
```

Comment:
Test methods should have `Test` in their names. You should change this to `ListAllTest`.